.github/workflows/ci.yml

```diff
@@ -42,6 +43,12 @@ jobs:
   tests:
     name: Integration Tests
     runs-on: ubuntu-latest
+    strategy:
```

Comment:
I think it would be helpful to add a comment about the modulo strategy so we know what we're looking at in the future.

Reply:
Will add in a followup PR 👍